src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -701,9 +706,127 @@ void regclass_graph_Model(py::module m) {
+             )");
+    model.def(
+        "get_rt_info",
+        [](const ov::Model& self, const py::list& path) -> py::object {
+            std::vector<std::string> cpp_args;
+            for (size_t i = 0; i < path.size(); i++) {
+                cpp_args.emplace_back(path[i].cast<std::string>());
+            }
+            return Common::utils::from_ov_any(self.get_rt_info<ov::Any>(cpp_args));
+        },
+        py::arg("path") = py::list(),
+        R"(
+                Returns runtime attribute.
+
+                :param path: List of strings which defines a path to runtime info.
+                :type path: List[str]
+
+                :return: A py::object
+                :rtype: py:object
+             )");
+    model.def(
+        "get_rt_info",
+        [](const ov::Model& self, const py::str& path) -> py::object {
+            return Common::utils::from_ov_any(self.get_rt_info<ov::Any>(path.cast<std::string>()));
+        },
+        py::arg("path") = py::str(),
+        R"(
+                Returns runtime attribute.
+
+                :param path: List of strings which defines a path to runtime info.
+                :type path: str
+
+                :return: A py::object
+                :rtype: py:object
+             )");
+    model.def(
+        "has_rt_info",
+        [](const ov::Model& self, const py::list& path) -> bool {
+            std::vector<std::string> cpp_args;
+            for (size_t i = 0; i < path.size(); i++) {
+                cpp_args.emplace_back(path[i].cast<std::string>());
+            }
+            return self.has_rt_info(cpp_args);
+        },
+        py::arg("path") = py::list(),
+        R"(
+                Checks if given path exists in runtime info of the model.
+
+                :param path: List of strings which defines a path to runtime info.
+                :type path: List[str]
+
+                :return: `True` if path exists, otherwise `False`.
+                :rtype: bool
+             )");
+    model.def(
+        "has_rt_info",
+        [](const ov::Model& self, const py::str& path) -> bool {
+            return self.has_rt_info(path.cast<std::string>());
+        },
+        py::arg("path") = py::str(),
+        R"(
+                Checks if given path exists in runtime info of the model.
+
+                :param path: List of strings which defines a path to runtime info.
+                :type path: str
+
+                :return: `True` if path exists, otherwise `False`.
+                :rtype: bool
+             )");
+    model.def(
+        "set_rt_info",
+        [](ov::Model& self, const py::object& obj, const py::list& path) -> void {
+            std::vector<std::string> cpp_args;
+            for (size_t i = 0; i < path.size(); i++) {
+                cpp_args.emplace_back(path[i].cast<std::string>());
+            }
+            self.set_rt_info<ov::Any>(py_object_to_any(obj), cpp_args);
+        },
+        py::arg("obj"),
+        py::arg("path") = py::list(),
```

Comment:
(*) - continuing the comment:

In my opinion, `path` shouldn't be empty, it will be overriding the same space `""` everytime the path is not passed.